### ISSUE TITLE
Mark standalone `implements \Traversable` erroneous

### ISSUE THREAD

**Issue by afoeder** [kalessil/phpinspectionsea#680]
Using
```
class Foo implements \Traversable
```
![image](https://user-images.githubusercontent.com/1125168/33655595-c4a33008-da73-11e7-9bf3-633a47481571.png)

should be marked erroneous since implementing Traversable in userland is not possible, see also [PHP doc on Traversable interface](http://php.net/manual/en/class.traversable.php)):

> Abstract base interface that cannot be implemented alone.
> [...]
> This is an internal engine interface which cannot be implemented in PHP scripts.

It might be checked if the candidate class is abstract and if it has inheritants who implement Iterator(Aggregate) which make the whole thing valid again 
// edit: won't work as well which simplifies the inspection I guess, https://3v4l.org/haeFh

**Comment by kalessil** [comment]
That's a sneaky case, picking up for EA Ultimate. Thank you @afoeder !

**Comment by kalessil** [comment]
For myself, test cases: https://3v4l.org/ciFk0

**Comment by kalessil** [comment]
- Throwable: 
```php
Note:
PHP classes cannot implement the Throwable interface directly, and must instead extend Exception.
```

**Comment by kalessil** [comment]
Implemented in https://github.com/kalessil/phpinspectionsea/commit/f46a2f9521a73dc181db253647eef0f806c6e21c
